app/mailers/patient_mailer.rb

```diff
@@ -207,6 +210,22 @@ def closed_email(patient)
     end
   end
 
+  def closed_sms(patient)
+    if patient&.primary_telephone.blank?
+      add_fail_history_closed_blank_field(patient, 'primary phone number')
+      return
+    end
+    if patient.blocked_sms
+      add_fail_history_sms_blocked(patient)
+      return
+    end
+
+    lang = patient.select_language
+    contents = I18n.t('assessments.sms.closed.thank-you', initials_age: patient&.initials_age('-'), locale: lang)
+
+    TwilioSender.send_sms(patient, contents)
```

Comment:
With the changes that were merged into 1.28.0 last night, you'll have to change `contents` to `[contents]`

Reply:
this should be resolved now